src/components/UserStats/UserStats.js

```diff
@@ -12,14 +12,14 @@ class UserStats extends React.Component {
 
   loadingBar = React.createRef();
 
-  baseUrl = `${process.env.REACT_APP_API_BASE_URL}/users`;
-
   retrieveStats = async (username) => {
+    const baseUrl = `${process.env.REACT_APP_API_BASE_URL}/users`;
```

Comment:
I kinda find this useless, you already have a base url, why creating a new one by just adding a slash/whatever? why not doing the whole thing straight on the api request? Again this is a matter of personal preference, theres nothing wrong with it

Reply:
this is ok in my opinion, it's not bad or good, it's just an opinion, the only bad thing is if you are not consistent with it

Reply:
I tried it out and I do think I like it better putting the whole thing in the api request. I originally had it set up like this because of doing multiple api requests in one file, but now I've changed that so it makes more sense to do it the other way. Thanks!